app/src/main/res/values/strings.xml

```diff
@@ -373,6 +373,21 @@
     <string name="note" comment="Users can add a note to an entry">Note</string>
     <string name="clear">Clear</string>
 
+    <string name="dialog_duplicate_entry_title">Duplicate Entry</string>
```

Comment:
Do we want "Entry" in upper or lower case? I think we usually do lower case throughout the app.